Hydro/src/Hydro/Platform/Vulkan/VulkanSwapChain.cpp

```diff
@@ -323,8 +324,17 @@ namespace Hydro
 
 		vkCmdBeginRenderPass(commandBuffer, &renderPassInfo, VK_SUBPASS_CONTENTS_INLINE);
 
-			Renderer3D::DrawMesh();
-			Renderer3D::End();
+			glm::mat4 transform2 = glm::mat4(1);
```

Comment:
Can be removed should be handled from the scene that calls the specific rendering functions based on the sprite component,